### DISCUSSION TITLE
Disable Tree Expansion when Dropping a New Tab into them

### DISCUSSION THREAD

**Discussion by Violetberry** [piroor/treestyletab#3348] [Q&A] [answered]
Hello everyone,
I'm looking for a way to prevent trees from automatically expanding when a new tab is dropped into them. Ideally, it would behave like dropping files into a folder. I've searched through the advanced settings and extensions but haven't found anything that addresses this specific issue.
Any help or guidance is appreciated.

**Comment by piroor** [comment, marked as answer]
[TST Lock Tree Collapsed](https://addons.mozilla.org/firefox/addon/tst-lock-tree-collapsed/) allows you to keep some parent tabs collapsed, so it can be a solution. Sadly it does not match to your idea exactly...

**Reply by Violetberry** [reply]
Well, then I must have expressed myself poorly because this works exactly like I wanted it to.
Thank you very much.
